lib/cogctl/action_util.ex

```diff
@@ -78,12 +78,21 @@ defmodule Cogctl.ActionUtil do
     :ok
   end
 
-  def display_error(errors) when is_list(errors) do 
-    IO.puts(:stderr, "ERROR: " <> Enum.join(errors, "\n"))
+  def display_warning(warnings) when is_list(warnings) do
+    Enum.map(warnings, &display_warning/1)
```

Comment:
What kind of weird things @vanstee. I haven't noticed anything... well I haven't noticed anything weirder than usual. I actually prefer to have it print one message at a time, but if it will cause issues I can switch it back.